src/bundle.c

```diff
@@ -723,6 +728,35 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	to_install_files = filter_out_existing_files(to_install_files);
 	grabtime_stop(&times);
 
+	/* Step 1.5: Now that we know the bundle is valid, check if we have enough space */
+	ptr = list_head(to_install_bundles);
```

Comment:
The comment suggests that all this loop does is calculate the space. In that case should we skip it if the skip-diskspace-check flag is set?

If on the other hand it is doing more, in particular making a deep clone of the bundles, then the comment ought to make it clear that it is doing both.

Reply:
You're right - this step should be skipped if the flag is set.
Indeed, all this loop does is calculate the space requirements.
No sense in calculating something and not using it.

Will change this.